packages/curvenote-cli/src/directives/listing.ts

```diff
@@ -0,0 +1,71 @@
+import type { DirectiveData } from 'myst-common';
+import type { VFile } from 'vfile';
+
+export const listingDirective = {
+  name: 'listing',
```

Comment:
Maybe `list:articles` as the type? Thinking we might want other listings in the future. Or `cn:articles`? As these are curvenote directives.